challenges/shakespearian-insult-generator.js

```diff
@@ -17,3 +17,27 @@ var second_word = ["weather-bitten", "unchin-snouted", "toad-spotted", "tickle-b
 var third_word = ["wagtail", "whey-face", "vassal", "varlet", "strumpet", "skainsmate", "scut", "ratsbane", "pumpion", "puttock", "pignut", "pigeon-egg", "nut-hook", "mumble-news", "moldwarp", "miscreant", "minnow", "measle", "mammet", "malt-worm", "maggot-pie", "lout", "lewdster", "joithead", "hugger-mugger", "horn-beast", "hedge-pig", "harpy", "haggard", "gudgeon", "giglet", "fustilarian", "foot-licker", "flirt-gill", "flax-wench", "flap-dragon", "dewberry", "death-token", "codpiece", "coxcomb", "clotpole", "clack-dish", "canker-blossom", "bum-bailey", "bugbear", "boar-pig", "bladder", "barnacle", "baggage", "apple-john"];
 
 // YOUR CODE HERE
+
+
+let generateInsult = function() {
+var result;
+let randOne = first_word[(Math.random() * first_word.length) | 0];
+let randTwo = second_word[(Math.random() * second_word.length) | 0];
+let randThree = third_word[(Math.random() * third_word.length) | 0];
+
+let askName = prompt("Who is this scoundrel that needs insulting?", "Enter a name");
```

Comment:
It makes sense you needed to add the HTML page to get these prompts to work--nice job going beyond the directions to complete the bonus.